sdk/identity/identity/src/credentials/interactiveBrowserCredentialOptions.ts

```diff
@@ -25,76 +25,71 @@ export type InteractiveBrowserAuthenticationFlow = "implicit-grant" | "auth-code
 /**
  * Defines the common options for the InteractiveBrowserCredential class.
  */
-export interface InteractiveBrowserCredentialCommonOptions extends TokenCredentialOptions {
-  /**
-   * Gets the redirect URI of the application. This should be same as the value
-   * in the application registration portal.  Defaults to `window.location.href`.
-   */
-  redirectUri?: string | (() => string);
-
-  /**
-   * Gets the URI to which the user will be redirected when logging out.
-   * Defaults to `window.location.href`.
-   */
-  postLogoutRedirectUri?: string | (() => string);
-
-  /**
-   * The Azure Active Directory tenant (directory) ID.
-   */
-  tenantId?: string;
+export type InteractiveBrowserCredentialOptions = TokenCredentialOptions &
+  InteractiveCredentialOptions & {
+    /**
+     * Gets the redirect URI of the application. This should be same as the value
+     * in the application registration portal.  Defaults to `window.location.href`.
+     */
+    redirectUri?: string | (() => string);
 
-  /**
-   * Result of a previous authentication that can be used to retrieve the cached credentials of each individual account.
-   * This is necessary to provide in case the application wants to work with more than one account per
-   * Client ID and Tenant ID pair.
-   *
-   * This record can be retrieved by calling to the InteractiveBrowserCredential's `authenticate()` method, as follows:
-   *
-   *     const authenticationRecord = await credential.authenticate();
-   *
-   */
-  authenticationRecord?: AuthenticationRecord;
+    /**
+     * Gets the URI to which the user will be redirected when logging out.
+     * Defaults to `window.location.href`.
+     */
+    postLogoutRedirectUri?: string | (() => string);
```

Comment:
We don't have this option in other languages that I'm aware of. When would it be used, given that we don't have any `Logout` APIs?

Reply:
I'll remove it!